README.md

```diff
@@ -1,8 +1,98 @@
-This tool takes timestamped images as an input and produces an animation
-that best fits in an allocated byte budget.
+# Thumbnailer
 
-You need to install the following:
+Thumbnailer takes timestamped images as an input and produces an animation in WebP format that best fits under a given size constraint.
 
-```sh
-sudo apt install libprotobuf-dev libprotoc-dev protobuf-compiler
+## Prerequisites
+
+- [WebP Library](https://github.com/webmproject/libwebp)
```

Comment:
Is libwebp really needed? It seems it's being built by bazel

Reply:
Yeah, it's supposed not to be needed but the CI would run failed if libwebp was not built dependently. We are looking into this issue.